dwave/embedding/chimera.py

```diff
@@ -137,6 +137,66 @@ def find_biclique_embedding(a, b, m, n=None, t=None, target_edges=None):
     return dict(zip(anodes, left)), dict(zip(bnodes, right))
 
 
+def find_grid_embedding(dim, m, n=None, t=4):
```

Comment:
Maybe in the future we could have a "constants" file that we could import. That way we can centralise all our default values in one place and easily update things should defaults need to be changed.

Ex.  DEFAULT_CHIMERA_SHORE_SIZE = 4